it/topics/scripting_languages/JavaScript/README.md

```diff
@@ -0,0 +1,224 @@
+    - il JavaScript eseguito internamente da jQuery funziona perfettamente su tutti i browser (che hanno ciascuno la propria implementazione di come funzionano le cose in JS). Quindi non devi preoccuparti di questo.
+    - [Pagina ufficiale](https://jquery.com/)
+        -> The Write Less, Do More, Libreria JavaScript
+        - La homepage mostra esempi di utilizzo.
+    - [Wikipedia](https://en.wikipedia.org/wiki/JQuery)
+        -> jQuery è una libreria JavaScript progettata per semplificare l'attraversamento e la manipolazione dell'albero HTML DOM, nonché la gestione degli eventi, l'animazione CSS e Ajax.
+    - [Scarica](https://jquery.com/download/).
+    - [API](https://api.jquery.com/) nei documenti.
+    - Tutorial
+        -  W3 Schools
+            - [Inizia](https://www.w3schools.com/jquery/jquery_get_started.asp).
+            - [jQuery](https://www.w3schools.com/jquery/default.asp).
+            - [Tutorial](https://www.w3schools.com/jquery/). La barra laterale copre i casi d'uso comuni.
+        - [Domande frequenti su jQuery](https://www.tutorialrepublic.com/faq.php#javascript-jquery) su un sito Web tutorial
+- ListJS
+    - [listjs.com](https://listjs.com)
+        > JavaScript vaniglia piccolo, invisibile e semplice, ma potente e incredibilmente veloce che aggiunge ricerca, ordinamento, filtri e flessibilità a semplici elenchi HTML, tabelle o altro.
+    - Aggiungi ricerca, ordina e filtra per la tua pagina HTML.
+
+
+### Librerie di template
+
+- Moustache JS
+    - [Pagina ufficiale](https://mustache.github.io/)
+        > Modelli senza logica.
+    - Modello HTML sul lato server o browser.
+    - [npm](https://www.npmjs.com/package/mustache)
+    - [Cloudfare CDN](https://cdnjs.com/libraries/mustache.js/)
+- Handlebars JS
+    - [Pagina ufficiale](https://handlebarsjs.com/)
+        > Templating minimo sugli steroidi
+    - [npm](https://www.npmjs.com/package/handlebars)
+    - [Github repo](https://github.com/wycats/handlebars.js)
+
+### Librerie Dataviz
+
+- [D3.js](https://d3js.org/)
+- [ApexCharts.js](https://apexcharts.com/)
+- [Highcharts](https://www.highcharts.com/)
+- roughViz
+    - [jwilber / roughViz](https://github.com/jwilber/roughViz) su Github.
+    - [roughviz](https://pypi.org/project/roughviz/) come pacchetto _Python_.
+    - [Tutorial](https://medium.com/better-programming/visualize-your-data-with-hand-drawn-charts-with-the-roughviz-charting-library-a3a9f3ee4f84) post del blog.
+
+### Frameworks
+
+Vedi il documento [JS Frameworks](frameworks.md) in questo progetto per i framework JS frontend e backend.
+
+## CDN
+
+Reti per la distribuzione di contenuti ([Wikipedia](https://en.wikipedia.org/wiki/Content_delivery_network)) dove è possibile scaricare le librerie.
+
+- CloudFare
+    - Pagina iniziale [cdnjs.com/ lasting(https://cdnjs.com/)
```

Comment:
@alessiobenvenuti Please preview as markdown. There are broken links in the CDN section